plugins/crd/handler/externalinterface/external_interface_handler.go

```diff
@@ -114,3 +115,61 @@ func (h *Handler) nodeInterfaceToProto(nodeIf v1.NodeInterface) *model.ExternalI
 	}
 	return protoVal
 }
+
+// Validation generates OpenAPIV3 validator for custom network CRD
+func Validation() *apiextv1beta1.CustomResourceValidation {
```

Comment:
change "validator for custom network CRD" to "validator for external interfaces"